subaru_outback_2016_eyesight.dbc

```diff
@@ -36,17 +36,32 @@ BS_:
 BU_: XXX
 
 
+BO_ 884 DoorStatus: 8 XXX
+ SG_ DoorOpenFD : 24|1@0+ (1,0) [0|1] "Front Driver Door" XXX
+ SG_ DoorOpenFP : 25|1@0+ (1,0) [0|1] "Front Passenger Door" XXX
+ SG_ DoorOpenRP : 26|1@0+ (1,0) [0|1] "Rear Passenger Door" XXX
+ SG_ DoorOpenRD : 27|1@0+ (1,0) [0|1] "Rear Driver Door" XXX
+ SG_ DoorOpenHatch : 28|1@0+ (1,0) [0|1] "Rear Hatch" XXX
```

Comment:
I believe it's good practice to just put the unit within quotes. In this case the signal would be unitless ("").
You can add comment separately at the bottom of the file (see other dbcs as examples)... but in this case the signal name itself is very self-explanatory.